### ISSUE TITLE
Weird error message for 'node describe'

### ISSUE THREAD

**Issue by aronchick** [bacalhau-project/bacalhau#2914]
```
❯ bacalhau node describe QmXZhend
could not get node QmXZhend: Unexpected response code: 400 ({
  "error": "code=400, message=failed to parse peer ID: length greater than remaining number of bytes in buffer",
  "message": "failed to parse peer ID: length greater than remaining number of bytes in buffer"
})
```

**Comment by simonwo** [comment]
I interpreted this as "the id you requested was not long enough". I think the best (UX) fix is to support short IDs, especially as that's the output from `node list`.

**Comment by simonwo** [comment]
Resolved by #2986.
